domain/model/bootstrap/bootstrap.go

```diff
@@ -0,0 +1,34 @@
+// Copyright 2023 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package bootstrap
+
+import (
+	"context"
+	"database/sql"
+	"fmt"
+
+	"github.com/juju/juju/core/database"
+	"github.com/juju/juju/domain/model"
+	"github.com/juju/juju/domain/model/state"
+)
+
+// Create is responsible for making a new model with all of it's associated
+// metadata during the bootstrap process.
+func CreateModel(
+	uuid model.UUID,
+	args model.ModelCreationArgs,
+) func(context.Context, database.TxnRunner) error {
+	return func(ctx context.Context, db database.TxnRunner) error {
+		if err := args.Validate(); err != nil {
+			return fmt.Errorf("bootstrap model creation args: %w", err)
```

Comment:
This should be "controller model creation args:"

Reply:
Why? There is nothing controller specific here. If it's related to the controller then caller can annotate that further.

Reply:
The model created at bootstrap time IS the "controller" model - literally. That's its name. It's not the "bootstrap" model. "bootstrap" is the operation we are performing, and the end result is that the model called "controller" is created.

Reply:
Have removed the work "bootstrap"